server/bin/pbench-server-prep-shim-002.sh

```diff
@@ -11,6 +11,11 @@
 # load common things
 . $dir/pbench-base.sh
 
+# need /usr/sbin in the PATH for restorecon
+if [[ ! ":$PATH:" =~ ":/usr/sbin:" ]]; then
+   PATH=$PATH:/usr/sbin; export PATH
+fi
+
```

Comment:
I guess the local add is fine if we never run into anything else from /usr/sbin. Although if selinux pollutes anything else, it might be cleaner to add it to the default path.

Reply:
I considered adding it to the trampoline script but in the end I thought it would be better to localize it as much as possible. Given that the shell script is going away in 0.70.0 (and we have to worry about the selinux python package, rather than the restorecon executable), I think I'll leave it as is.